Resources/views/Form/fields.html.twig

```diff
@@ -336,11 +337,11 @@
   {%- endif -%}
   {%- set widget_attr = {} -%}
   {%- if help is not empty -%}
-    {%- set widget_attr = {attr: {'aria-describedby': id ~"_help"}} -%}
+    {%- set widget_attr = {attr: {'aria-describedby': id ~ "_help"}} -%}
   {%- endif -%}
   <{{ element|default('div') }} class="form-group">
```

Comment:
Dit versta ik niet zo goed, waarom is die `<` weg?

Reply:
Er is toch geen < weg, de change is de spatie die is toegevoegd: https://www.evernote.com/l/AAaMC0LtvEtKCI3diXfp7PvS7cqMl5EkytE